vkcs/compute/data_source_flavor.go

```diff
@@ -102,6 +103,13 @@ func DataSourceComputeFlavor() *schema.Resource {
 				Description: "The flavor visibility.",
 			},
 
+			"cpu_generation": {
```

Comment:
@ftersin  We already have `extra_specs` attribute, which is computed now, can't we reuse it and make it optional to specify `mcs:cpu_generation` there? We can't do this with maps in resources, if I remember correctly, as, I believe, overwritten map would lead to inconsistent result on apply or non-empty plan, but it should be ok in data source.

Reply:
Add ability to set extra_specs

Reply:
No, I think it's ok, for now.